examples/tutorial-standalone/variables.tf

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "org_id" {
+  description = "The numeric organization id."
+}
+
+variable "folder_id" {
+  description = "The folder to deploy in."
+}
+
+variable "billing_account" {
+  description = "The billing account id associated with the projects, e.g. XXXXXX-YYYYYY-ZZZZZZ."
+}
+
+variable "access_context_manager_policy_id" {
+  description = "The id of the default Access Context Manager policy. Can be obtained by running `gcloud access-context-manager policies list --organization YOUR-ORGANIZATION_ID --format=\"value(name)\"`."
+  type        = string
+  default     = ""
```

Comment:
I thought we needed a policy defined as required in the main - I don't think defaulting this is useful to hide unless we're showing how to make a policy

Reply:
Currently if the user do not send a `access_context_manager_policy_id` the main module will create to him.

 https://github.com/GoogleCloudPlatform/terraform-google-secured-data-warehouse/blob/af9a522f2e5e4cf0fcec469269aaf4ef315c0e40/service_control.tf#L71-L74
 
In the foundation we have the `create_access_context_manager_access_policy` variable with `true` as default and in the [README](https://github.com/terraform-google-modules/terraform-example-foundation/tree/master/1-org#deploying-with-cloud-build) we recommend to the user put `false` if he already have this value. 

Should the data secured warehouse should create the `access_context_manager_policy_id`? If yes, should we implement as the same way that the foundation?

Reply:
oh interesting.  it's fine that we make an ACM policy for the data warehouse.  The default behavior matches.  

I guess the main point is for the example/tutorial, any reason to have a variable (in the example), when we want to be prescriptive where the choice is we make a policy (i.e the main module)?

Reply:
We discussed it internally and for now we decided to remove this variable from `example/standalone`.

Reply:
got it.  change looks good